// Copyright 2015 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

import "point.proto";

option optimize_for = LITE_RUNTIME;

package cc.proto;

enum SelectionBoundType {
  UNKNOWN = 0;
  LEFT = 1;
  RIGHT = 2;
  CENTER = 3;
  EMPTY = 4;
}

message LayerSelectionBound {
  optional SelectionBoundType type = 1;
  optional Point edge_top = 2;
  optional Point edge_bottom = 3;
  optional int32 layer_id = 4;
}
